source/retryable-writes/tests/bulkWrite.yml

```diff
@@ -294,7 +294,7 @@ tests:
             error: true
             result:
                 deletedCount: 0
-                insertedIds: { 0: 2 }
+                insertedIds: { 1: 2 }
```

Comment:
Drivers that don't report `insertedIds` in their bulk write result (e.g. libmongoc) would not have caught this mistake.